waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/context/CommonBeans.java

```diff
@@ -39,7 +39,7 @@
   public HiveConf hiveConf(WaggleDanceConfiguration waggleDanceConfiguration) {
     Map<String, String> confProps = waggleDanceConfiguration.getConfigurationProperties();
 
-    final HiveConf hiveConf = new HiveConf(new Configuration(false), getClass());
+    final HiveConf hiveConf = new HiveConf(new Configuration(), getClass());
```

Comment:
If I am not mistaken this change means the configuration will load the hive-site.xml from classpath or default locations. I am not sure we want that in general it might pull in properties we don't want. Generally though we run WD on separate instances where hive is not installed. Would it work for you to add the Hive Kerberos configuration properties in the WD yaml files as they are loaded into the hiveConf in this method.
You can add the properties in the waggle-dance-server.yml: `configuration-properties`. This is undocumented apologies we should fix that.

Reply:
Could you explain which class stores yaml propeties? WaggleDanceConfiguration? 
We'll make loading required properties from yml, but it makes sense to keep the ability to load them from native hive-site.xml. F.e. via additional yml property like "use-native-config"

Reply:
yes `WaggleDanceConfiguration.getConfigurationProperties()`
Please add the loading via native in a separate PR, I'd prefer to keep this PR focused on adding Kerberos support only if possible.